src/test/java/uk/gov/justice/digital/hmpps/cmd/api/service/NotificationServiceTest_Generate_Shift.kt

```diff
@@ -77,13 +77,13 @@ internal class NotificationServiceTest_Generate_Shift {
       val task = "Guard Duty"
       val shiftType = ShiftType.SHIFT
       val dto1 = CsrModifiedDetailDto(
-        quantumId,
-        today,
-        shiftType,
-        start,
-        end,
-        task,
-        DetailModificationType.ADD
+        quantumId = quantumId,
+        shiftModified = today,
+        shiftType = shiftType,
+        detailStart = start,
+        detailEnd = end,
+        activity = task,
+        actionType = DetailModificationType.ADD
```

Comment:
I was going to edit subsequent occurrences like this in a similar way, but realised this test class will eventually get thrown away in favour of the dryrun version